src/error.rs

```diff
@@ -50,6 +50,16 @@ impl From<validator::ValidationErrors> for Error {
 
 impl ResponseError for Error {
     fn error_response(&self) -> HttpResponse {
-        HttpResponse::new(StatusCode::BAD_REQUEST)
+        HttpResponse::build(StatusCode::BAD_REQUEST).body(match self {
+            Self::Validate(e) => format!(
+                "{{\"message\": \"invalid {}\" }}",
```

Comment:
Maybe better to use plain text for error reporting by default like actix-web?

Reply:
so just the errors joined with , is fine?

Reply:
I think yes)